railties/lib/rails/secrets.rb

```diff
@@ -30,20 +32,24 @@ def parse(paths, env:)
       end
 
       def generate_key
-        cipher = new_cipher
-        SecureRandom.hex(cipher.key_len)[0, cipher.key_len]
+        new_cipher.random_key
```

Comment:
This is random bytes, right?

If so, I think we should probably pack/unpack to hex, to make it safer for people to move around.